users/tests.py

```diff
@@ -78,3 +78,37 @@ def test_create_user_with_long_name(self):
                 last_name="last_name",
                 role=Role.EMPLOYEE,
                 team=TestTeams.valid_team)
+
+    @pytest.fixture
+    def example_team(self):
+        team = Team(name='a', description='b')
+        team.save()
```

Comment:
```suggestion
        team = Team.objects.create(name='a', description='b')
```
objects.create already saves the object